Linguini.Syntax.Tests/Linguini.Syntax.Tests.csproj

```diff
@@ -1,13 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-
         <IsPackable>false</IsPackable>
-
         <Nullable>enable</Nullable>
-
-        <TargetFrameworks>net5.0;netstandard2.1</TargetFrameworks>
-
+        <TargetFrameworks>net4.6.1;net5.0;netstandard2.1</TargetFrameworks>
```

Comment:
The other files use `net461`, and just this one uses `net4.6.1`. Probably doesn't matter, but just pointing it out in case.